subsys/secure_storage/src/aead/aead_key_huk.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <hw_unique_key.h>
+#include <nrf_cc3xx_platform.h>
+#include <zephyr/init.h>
+
+#include "aead_key.h"
+
+#ifdef HUK_HAS_KMU
+#define KEYSLOT HUK_KEYSLOT_MKEK
```

Comment:
Is MKEK or MEXT the right one here? MEXT says external storage, which seems like what PS is for.

Reply:
I just copied that code from our sample https://github.com/nrfconnect/sdk-nrf/blob/main/samples/keys/hw_unique_key/src/derive_key.c#L15 

`_Master key encryption key (MKEK). It is used for deriving Key Encryption Keys (KEKs), which are used to encrypt other keys when these are stored. It is provided to CryptoCell when it is used._`
As this storage was intended to be used mainly to store keys MKEK seems all right